backend/Readme.md

```diff
@@ -0,0 +1,29 @@
+# CleanAirMap Backend
+
+## About
+The CleanAirMap backend is a GraphQL schema and data storage for the client app. The schema is implemented using GraphQL Nexus, and the primary data model is managed by Prisma Migrate
+
+## Getting started
+To try out the backend, you can get an example running using Docker Compose
+
+1. Clone the repository
+2. From the respository root:
+   1. `npm install` to get the dependencies
+   2. `npm run build` to build the project
+   3. Configure the service using the .env file. You can use env.sample as a guide
+   4. `docker-compose up` to build the server image and spin up the service
+
+
+## Making changes
+
+While working on the GraphQL implementation, it is recommended to have the development server running. It will detect changes and reload, generating the required Typescript types on-the-fly.
+
+To run the development server
+
+1. Have a PostGres listening on the DATABASE_PORT defined in .env
```

Comment:
You might want to call it an instance and not capitalize the G
_Have a Postgres instance listening..._

Reply:
Good call